This could be a comment about the patch itself, where we could use an
Index: a/ header with /../ inside that could be interpreted as a	
malicious pseudo-header, so we should not validate it,

--- /dev/null
+++ b/inert-file
@@ -0,0 +1,1 @@
+Inert
